test/fixtures/mysql-ha/outputs.tf

```diff
@@ -15,13 +15,14 @@
  */
 
 output "project_id" {
-  value = "${var.project}"
+  value = var.project
```

Comment:
descirpiton?

Reply:
Not in outputs, because of variables.tf
#####################
"project" {
  type        = string
  description = "The project to run tests against"
}
######################################

Reply:
I've added it